apps/python-testsgk/python_testsgk/app.py

```diff
@@ -0,0 +1,104 @@
+from flask import Flask, jsonify, request
+import sqlite3 as sl
+
+
+app = Flask(__name__)
+conn = sl.connect("tasks.db")
+curs = conn.cursor()
+
+stmtCreateDB = (
```

Comment:
Python naming is in snake_case not camelCase for variables and functions
```suggestion
stmt_create_db = (
```